# parameters for script/motion.py in outdoor mode

control_rate : 1.0

navigation_approach_margin: [0.3, 0.3, 0.3]

nav_mode: 5
search_height : 2.0
search_grid_size : 0.00001
reach_margin : 0.5
target_timeout : 10.0
approach_height : 1.5
descending_height : 1.2
approach_margin : 0.15
height_margin : 0.15
cover_pose: [0.8, 0.8] # the joint angles for covering
covering_pre_height : 1.2
covering_post_height : 1.2
covering_move_dist : 0.6

close_pose: [1.56, 1.56] # the joint angles  for closing

search_area_number: 1
initial_waypoints: [[[49.9, 8.9, 2]]]
area_corners : [[[49.9, 8.9], [49.90005, 8.89995], 20]]
